aioseop_class.php

```diff
@@ -126,7 +126,7 @@ function __construct() {
 				__( '%blog_description% - Your blog description', 'all-in-one-seo-pack' ) . '</li><li>' .
 				__( '%post_title% - The original title of the post', 'all-in-one-seo-pack' ) . '</li><li>' .
 				__( '%category_title% - The (main) category of the post', 'all-in-one-seo-pack' ) . '</li><li>' .
-				__( '%category% - Alias for %category_title%', 'all-in-one-seo-pack' ) . '</li><li>' .
+				__( '%1$category% - Alias for %2$category_title%', 'all-in-one-seo-pack' ) . '</li><li>' .
```

Comment:
This appears to be a Change. Did you add this by accident?

Reply:
this is done by grunt and it follows the standards of http://php.net/manual/en/function.sprintf.php - look at "Example #3 Argument swapping". This is in fact the correct way of using any sprintf-based function.

Reply:
Where is the `sprintf()`?!